src/Google/Service/Resource.php

```diff
@@ -272,6 +272,9 @@ public function createRequestUri($restPath, $params)
     $uriTemplateVars = array();
     $queryVars = array();
     foreach ($params as $paramName => $paramSpec) {
+      if (gettype($paramSpec) !== 'array') {
```

Comment:
I'm interested to see that you started with `is_array` before switching. What was the issue with `is_array`?

Reply:
Personally i like the `is_array` over the `gettype`, but i actually didn't think that `is_array` was compatible with `php 5.4`

I can change it back to `is_array`

Reply:
`is_array` has been available since PHP 4. Definitely prefer it over `gettype`.